armi/cases/tests/test_cases.py

```diff
@@ -408,7 +413,12 @@ def test_buildCommand(self):
 
 
 class TestExtraInputWriting(unittest.TestCase):
-    """Make sure extra inputs from interfaces are written."""
+    """Make sure extra inputs from interfaces are written.
+
+    .. test:: Write blueprints settings to disk
+        :id: T_ARMI_BP_TO_DB1
+        :tests: R_ARMI_BP_TO_DB
```

Comment:
Similar to my last to comments, I think this test should be moved to `saveToStream()`.